lib/services/registered_addons.pm

```diff
@@ -32,7 +32,7 @@ sub check_registered_system {
     $pro = 'SLE_' . $pro if ($pro eq 'HPC');
     suseconnect_ls($pro);
     my $ver = $system =~ s/\-SP/./r;
-    script_run("SUSEConnect -s | grep " . $ver);
+    script_run("SUSEConnect -s | grep " . $ver, die_on_timeout => 0);
```

Comment:
I suppose this should be assert_script_run? 
Add this parameter can fix ticket or not?

Reply:
We use script_run originally will take this run as not important test, that's means we don't care the return value of this line and don't want to stop here and hope to continue run other test. Add die_on_timeout => 0 just make script_run to work as original expectation.